### PR TITLE
Bugfix: Local instance

### PR DESCRIPTION
**Pull request by nmasnadithya**

### CODE REVIEW

File: app/server/labml_app/utils/analytics.py [hunk 1]
```diff
@@ -70,7 +70,7 @@ def get_meta_data(self, request: Request) -> Dict[str, str]:
         return meta
 
     def track(self, request: Request, event: str, data: Union[NamedTuple, Dict], identifier: str = '') -> None:
-        if isinstance(data, NamedTuple):
+        if isinstance(data, type(NamedTuple)):
```
Comment (vpj) [inline app/server/labml_app/utils/analytics.py:73]: this looks wrong

File: app/Makefile [hunk 2]
```diff
@@ -60,6 +60,10 @@ clean: clean-pycache ## Cleanup builds
 package: build-ui clean-pycache ## Build PIPy Package
 	rm -rf server/labml_app/static
 	cp -r static/ server/labml_app/static
+	rm -f server/labml_app/analyses_settings.py
+	rm -f server/labml_app/block_uuids.py
+	rm -f server/labml_app/settings.py
+	cp server_static/*.py server/labml_app/
```
Comment (vpj) [inline app/Makefile:66]: Can you change `server_static` to something like `local_server_settings`? Thats more meaningful right?

File: app/server/labml_app/utils/analytics.py (hunk 1 above)
Comment (nmasnadithya) [reply app/server/labml_app/utils/analytics.py:73]: The following error occurs if it's not wrapped with `type()`

``` 
  File "/home/ck/miniconda/lib/python3.9/site-packages/labml_app/utils/analytics.py", line 107, in time_wrapper
    self.track(request, func.__name__, {'time_elapsed': str(total_time)})
  File "/home/ck/miniconda/lib/python3.9/site-packages/labml_app/utils/analytics.py", line 76, in track
    if isinstance(data, NamedTuple):
TypeError: isinstance() arg 2 must be a type or tuple of types
```

File: app/Makefile (hunk 2 above)
Comment (nmasnadithya) [reply app/Makefile:66]: Yeah, will change that

**Pull request merged**